intelmq/bots/experts/domain_valid/expert.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""
+Domain validator
+
+SPDX-FileCopyrightText: 2021 Marius Karotkis <marius.karotkis@gmail.com>
+SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+
+import validators
```

Comment:
Does this refer to https://pypi.org/project/validators/ ? if yes, please create a REQUIREMENTS.txt in this directory and add a note to the bot documentation